ndk-build/src/manifest.rs

```diff
@@ -232,30 +232,23 @@ pub struct MetaData {
     pub value: String,
 }
 
-/// https://developer.android.com/guide/topics/manifest/uses-feature-element
+/// Android [uses-feature element](https://developer.android.com/guide/topics/manifest/uses-feature-element).
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
 pub struct Feature {
     #[serde(rename(serialize = "android:name"))]
     pub name: Option<String>,
     #[serde(rename(serialize = "android:required"))]
     pub required: Option<bool>,
-    /// The `version` field is used for Vulkan specific hardware features. `version`
-    /// indicates different requirements based on the `name` field as desribed below:
+    /// The `version` field is currently used for the following features:
     ///
-    /// - name: "android.hardware.vulkan.compute", `version` indicates which level of
-    /// optional compute features the app requires beyond the Vulkan 1.0 requirements.
+    /// - `name="android.hardware.vulkan.compute"`: The minimum level of compute features required. See the [Android documentation](https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_COMPUTE)
+    ///   for available levels and the respective Vulkan features required/provided.
     ///
-    /// More detail: <https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_COMPUTE>
+    /// - `name="android.hardware.vulkan.level"`: The minimum Vulkan requirements. See the [Android documentation](https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_LEVEL)
+    ///   for available levels and the respective Vulkan features required/provided.
     ///
-    /// - name "android.hardware.vulkan.level", `version` indicates which level of
-    /// optional hardware features the app requires.
-    ///
-    /// More detail: <https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_LEVEL>
-    ///
-    /// - name "android.hardware.vulkan.compute", `version` indicates the minimum version
-    /// of Vulkan API support the app requires.
-    ///
-    /// More detail: <https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_VERSION>
+    /// - `name="android.hardware.vulkan.version"`: The minimum version of the Vulkan API required. See the [Android documentation](https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_VERSION)
+    ///    for available levels and the respective Vulkan features required/provided.
```

Comment:
Perhaps worth mentioning that this is a bitmask with the same encoding as Vulkan version numbers, representing the value of `VkPhysicalDeviceProperties::apiVersion` (and has nothing to do with Vulkan features in the sense as the other two features).